### PR TITLE
[CGPROD-2487] Config file changes

### PR DESCRIPTION
**Pull request by AdamXA**
fixes genie

### CODE REVIEW

File: src/components/overlays/pause.js [hunk 1]
```diff
@@ -18,9 +19,9 @@ export class Pause extends Screen {
         this.addBackgroundItems();
         const parentKey = this.context.parentScreens.slice(-1)[0].scene.key;
         const isAboveSelectScreen = parentKey.includes("select");
-        const achievements = this.context.config.theme.game.achievements ? ["achievements"] : [];
+        const achievements = gmi.achievements.get().length ? ["achievements"] : [];
```
Comment (FostUK) [inline src/components/overlays/pause.js:22]: I like this - so to disable achievements do you just supply an empty achievements config?

Comment (AdamXA) [reply src/components/overlays/pause.js:22]: yup

**Comment by AdamXA** [comment]
CI passes just PR plugin bugged

**Comment by AdamXA** [comment]
retest this please

**Pull request merged**